tests/integration/did.test.js

```diff
@@ -60,6 +60,32 @@ describe('DID Module', () => {
     expect(!!result).toBe(true);
   }, 10000);
 
+  test('Can update a DID controller', async () => {
+    const pair = dock.keyring.addFromUri(firstKeySeed);
+    const publicKey = getPublicKeyFromKeyringPair(pair);
+
+    // Get the current did controller
+    const originalDoc = await dock.did.getDocument(dockDID);
+    const currentController = getHexIdentifierFromDID(originalDoc.publicKey[0].controller);
+
+    // Send key update without changing controller
+    const [keyUpdateNone, signatureNone] = await createSignedKeyUpdate(dock.did, dockDID, publicKey, pair);
```

Comment:
The `None` in the name sounds odd. Can you use `let` in place of `const`?

Reply:
good point, was taken from andrew's comments in #102 but doesnt make much sense reading back

Reply:
also why use let in this case?